auth_tokens.ts

```diff
@@ -46,6 +46,8 @@ export class AuthTokens {
           tokens.push({ type: "bearer", host, token });
         }
       } else {
+        // todo(dsherret): feel like this should error?
+        // deno-lint-ignore no-console
```

Comment:
Is it better to just throw here?

Reply:
Yeah, that's what I added the todo about. It's weird this code does this. I opened https://github.com/denoland/deno_cache/issues/43 just now.